services/QuillLMS/client/app/bundles/Staff/components/evidence/configureRules/rulePlagiarismAttributes.tsx

```diff
@@ -97,12 +97,15 @@ const RulePlagiarismAttributes = ({
     if (plagiarismText._destroy) { return <span /> }
 
     return (
-      <PlagiarismTextEditor
-        index={i}
-        key={i}
-        setPlagiarismText={onHandleSetPlagiarismText}
-        text={plagiarismText.text}
-      />
+      <React.Fragment>
+        <PlagiarismTextEditor
+          index={i}
+          key={i}
+          setPlagiarismText={onHandleSetPlagiarismText}
+          text={plagiarismText.text}
+        />
+        {!plagiarismText.valid_in_all_targets && <p className="all-errors-message">This plagiarism text does not match the associated activity text. This means that it will not trigger the plagiarism feedback as intended. Please update the text above.</p>}
```

Comment:
One subtle difference between bad highlights and bad plagiarism text is that a badly defined highlight won't display at all, but a badly defined plagiarism text will still catch a lot of plagiarism as long as it's close.  I don't think that means we need to change the language here or anything, but it may be worth making sure that Curriculum understands that while highlight text that is close-but-not exact doesn't work at all, they may see that plagiarism mostly gets caught even with out-of-date texts.

We still want to flag this so that they can update them, especially in cases where the plagiarism text changes significantly, so I don't think I'm suggesting code changes, but we do want Curriculum to understand the distinction.

Reply:
This is a good point. I'm going to change this to say "may not trigger", and will drop an extra note to the curriculum team that it may or may not be necessary to update all of these. Some of the ones I noticed in sampling the data were the difference of a word or two.